django/contrib/admin/static/admin/js/admin/RelatedObjectLookups.js

```diff
@@ -157,4 +157,13 @@
     window.showAddAnotherPopup = showRelatedObjectPopup;
     window.dismissAddAnotherPopup = dismissAddRelatedObjectPopup;
 
-})();
+    $(document).ready(function() {
+        $("a[data-popup_opener]").each(function() {
```

Comment:
actually this is correct, there's no obvious common parent element for all of these to use so we'd have to put a listener on document.

Also new elements like these are only created from the backend